chords/once_again.onsong

```diff
@@ -27,9 +27,5 @@ And [G]I'm full of [A]praise once a[D]gain
 
 Bridge
 [G]Thank you for the [A]cross
-[G]Thank you for the [A]cross
-[G]Thank you for the [A]cross, my [D]friend
-
-[Bm]Thank you for the [C#m]cross
-[A]Thank you for the [D]cross
+[D/F#]Thank you for the [Bm]cross
```

Comment:
`[Bm]Thank` sounded really odd to me here. My suggestion here is from [Worship Together](https://www.google.com/search?client=ubuntu&channel=fs&q=worship+together+once+again&ie=utf-8&oe=utf-8)